(dp1
S'output'
p2
S"<type 'exceptions.KeyError'> 'id'"
p3
sS'layer'
p4
S'C:\\Users\\Pieter-Jan\\PycharmProjects\\untitled\\web2py\\applications\\Sensors\\controllers/overzicht.py'
p5
sS'code'
p6
S'# -*- coding: utf-8 -*-\n# probeer zoiets als\n\ndef index():\n    rows = db(db.sensor.id > 0).select()\n    return dict(message="hello from overzicht.py", rijen=rows)\n\n\ndef detail():\n    #print("detail() in overzicht.py")\n    sensorNaam = request.vars[\'naam\']\n    time = db(db.data.sensor == sensorNaam).select(db.data.tijd).as_list()\n    temp = db(db.data.sensor == sensorNaam).select(db.data.temperatuur).as_dict()\n\n    print temp\n\n    from bokeh.plotting import figure\n    from bokeh.resources import CDN\n    from bokeh.embed import file_html\n\n    plot = figure()\n    plot.line(x=time, y=temp)\n\n    html = file_html(plot, CDN, "my plot")\n    return dict(bokeh_script=html)\n\nresponse._vars=response._caller(detail)\n'
p7
sS'snapshot'
p8
(dp9
sS'traceback'
p10
S'Traceback (most recent call last):\n  File "C:\\Users\\Pieter-Jan\\PycharmProjects\\untitled\\web2py\\gluon\\restricted.py", line 227, in restricted\n    exec ccode in environment\n  File "C:\\Users\\Pieter-Jan\\PycharmProjects\\untitled\\web2py\\applications\\Sensors\\controllers/overzicht.py", line 27, in <module>\n  File "C:\\Users\\Pieter-Jan\\PycharmProjects\\untitled\\web2py\\gluon\\globals.py", line 417, in <lambda>\n    self._caller = lambda f: f()\n  File "C:\\Users\\Pieter-Jan\\PycharmProjects\\untitled\\web2py\\applications\\Sensors\\controllers/overzicht.py", line 13, in detail\n    temp = db(db.data.sensor == sensorNaam).select(db.data.temperatuur).as_dict()\n  File "C:\\Users\\Pieter-Jan\\PycharmProjects\\untitled\\web2py\\gluon\\packages\\dal\\pydal\\objects.py", line 2322, in as_dict\n    return dict([(r[key], r) for r in rows])\nKeyError: \'id\'\n'
p11
s.